po/da.po

```diff
@@ -10138,83 +10180,83 @@ msgstr ""
 
 #: src/bare-metal/no_std.md:25
 msgid "`NonZeroU8`..."
-msgstr ""
+msgstr "`NonZeroU8`..."
 
 #: src/bare-metal/no_std.md:26
 msgid "`Option`, `Result`"
-msgstr ""
+msgstr "`Option`, `Result`"
 
 #: src/bare-metal/no_std.md:27
 msgid "`Display`, `Debug`, `write!`..."
-msgstr ""
+msgstr "`Display`, `Debug`, `write!`..."
 
 #: src/bare-metal/no_std.md:29
 msgid "`panic!`, `assert_eq!`..."
-msgstr ""
+msgstr "`panic!`, `assert_eq!`..."
 
 #: src/bare-metal/no_std.md:30
 msgid "`NonNull` and all the usual pointer-related functions"
 msgstr ""
 
 #: src/bare-metal/no_std.md:31
 msgid "`Future` and `async`/`await`"
-msgstr ""
+msgstr "`Future` og `async`/`await`"
 
 #: src/bare-metal/no_std.md:32
 msgid "`fence`, `AtomicBool`, `AtomicPtr`, `AtomicU32`..."
-msgstr ""
+msgstr "`fence`, `AtomicBool`, `AtomicPtr`, `AtomicU32`..."
 
 #: src/bare-metal/no_std.md:33
 msgid "`Duration`"
-msgstr ""
+msgstr "`Duration`"
```

Comment:
Varighed?

Reply:
This is a Rust type, so it should not be translated.